### ISSUE TITLE
Change the orderby

### ISSUE THREAD

**Issue by moseleyi** [crudlio/crudl-example-express#12]
I'm trying to make the demo version working with my GraphQL endpoint. I managed to do everything apart from orderBy. By default it's called like:

`orderBy:"username"`

but I need to be able to change that to:

`orderBy:[username]
`
Where can I change it?

**Comment by sehmaschine** [comment]
https://github.com/crudlio/crudl-example-express/blob/master/blog/graphql/schema.js#L65

does that help?

**Comment by moseleyi** [comment]
Hmm I'm using GraphQL Sequelize sequelizeConnection that only let's me do this:

```
orderBy : { new GraphQL.GraphQLEnumType({
    name : "username",
    values : {
        "username" : {value : ['username', 'ASC']}
    }
}
```

but of course that means I have to reference it `orderBy:[username]` I had to hack the listView.js but I would prefer not to

**Comment by moseleyi** [comment]
Not exactly. GraphQL exposes a type `username` that can have a value `username`, which can only be called using `orderBy:[username]`

I'm working on some workarounds and better example to present some more information soon

**Comment by sehmaschine** [comment]
You can also take a look at the connectors (resp. the middleware):
https://github.com/crudlio/crudl-example-express/blob/master/crudl-admin-rest/admin/connectors/middleware/buildQuery.js#L16

Here you get the data from the form and build the API call (the connector is the gateway between the form and your API). I still think it´s strange to send an array to the API though.
